emails/templates/emails/first_time_user.html

```diff
@@ -0,0 +1,411 @@
+{% comment %}
+  Note that Django only loads strings from `app.ftl`, `brands.ftl`, and `phones.ftl`
+  so make sure all email strings are included in those files.
```

Comment:
`app.ftl` was refactored away, see the current list in `privaterelay/ftl_bundles.py`

```suggestion
  Note that Django only loads strings from `brands.ftl`, `layout.ftl`, `misc.ftl`, `phones.ftl`, and `privaterelay/pending_locales/en/pending.ftl`
  so make sure all email strings are included in those files.
```